Pipfile

```diff
@@ -4,28 +4,35 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-conan = {version="==1.59.0"}
 cookiecutter = {version="==2.1.1"}
 gcovr = "*"
 hammocking = "*"
 kconfiglib = "*"
 
 [dev-packages]
+jinja2 = "*"
```

Comment:
Copied all dependencies from SPLED to make sure the docs and spl-core python integration tests are working.
Keep in mind that the integration tests are using the spl-core python venv, that is why we have to add so many dev dependencies.